pkgs/tools/misc/h5utils/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchurl, hdf5, lib }:
+
+let
+  mpi = hdf5.mpi;
+  mpiSupport = hdf5.mpiSupport;
+in stdenv.mkDerivation rec {
+  version = "1.13";
+  name = "h5utils-${version}";
+
+  src = fetchurl {
```

Comment:
Could you please switch to `fetchFromGitHub`.

Reply:
Ok. I also think it's simpler to fetch this archive. So `fetchurl` is fine in this case.
Could you then just add a comment explaining this choice?